content/post/059-ipfs-camp-sci-fi-fair-videos.md

```diff
@@ -0,0 +1,69 @@
+
+![@achingbrain's npm-in-a-box](/059-ipfs-camp-sci-fi-fair-videos/npm-in-box.jpg)
+_@achingbrain's npm-in-a-box project_
+
+**BONUS:** If you like big IPFS nodes running on small computers (and/or "The IT Crowd"), you may also enjoy Alex's "npm in a box", which packs the entire NPM package repository on a very portable (and sleek looking, if we say so) RPi based IPFS node with a BIG archival grade hard drive. Take this with you on your field trip and your team will never see an npm timeout ever again.
+
+### MetaMask's Network Visualizer
+
+{{< youtube 7HDVgjQnNdM >}}
+_@kumavis shows off the live network visualizer_
+
+The MetaMask crew brought a tool that proved invaluable in their network connectivity diagnostics: a DHT visualizer. Following some wireless network trouble (a real-world scenario to be sure), we were treated to some beautiful visualizations of network connectivity.
+
+### And More
+These are just some of the projects that made it to the Fair: head over to the [offical Fair playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtTWCyJcTgLFhsg8Bi2dFPCo) to catch the rest of the videos, as well as the full cuts of the interviews above. Don't miss Berty's Bluetooth Low Energy p2p transport, Brave's one-click in-browser IPFS, and more!
+
+### Lest We Forget
+Our tireless camera crew could only be in one place at a time (we're working on it), which has left some of our favorite projects sadly underdocumented in the video department. Here are some stills that fill in the gaps:
+
+![@MarneeDear's FARS + @gorhgorh's minitel IPFS](/059-ipfs-camp-sci-fi-fair-videos/marnee-minitel.jpg)
+_Marnee Dearman's IPFS-over-Ham Radio FARS project and Jérôme's IPFS Chat on a French Minitel terminal_
+
+![@vasa-develop's 3D collaboration](/059-ipfs-camp-sci-fi-fair-videos/vasa.jpg)
+_Vasa's CRDT-based collaborative 3D modeling interface_
+
+![@gorhgorh's catbot](/059-ipfs-camp-sci-fi-fair-videos/catbot.jpg)
+_Jérôme Loï's libp2p-powered Lazer Catbot (unfortunately no cats were on premises)_
+
+![the elusive @rklaehn](/059-ipfs-camp-sci-fi-fair-videos/actyx.jpg)
+_Actyx's Rüdiger has somehow managed to avoid ALL photos at the fair, so here's a shot of him demoing their amazing industrial automation system from the lightning talks_ 
+
+![@aschmahmann's Fast IPNS](/059-ipfs-camp-sci-fi-fair-videos/ipns.jpg)
+_Last but not least, Adin's "Fast IPNS" demo (it was VERY FAST!)_
+
+## Community Interviews
+IPFS Camp was all about YOU so the interviews weren't limited to just the Sci-Fi Fair participants: we managed to get some async sit-down time with a few other wonderful community members: see their accounts in [this playlist](TK)
```

Comment:
IPFS Camp was all about YOU so the interviews weren't limited to just the Sci-Fi Fair participants: we managed to get some sit-down time with a few other wonderful community members to talk about their projects. Check out the videos below or skip right to the [playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtTwwGv30nKdhr3GiIAnhbyE). 

[Netflix Containers - Edgar Lee](https://youtu.be/wNfk05D887M) 
We hear from Edgar about how he's exploring distributing container layers through IPFS, and how it's changing his workflow at Netflix. 

[Audius - Hareesh Nagaraj](https://youtu.be/3pSV6ai6QUI) 
We speak to Hareesh from Audius, a decentralized music streaming platform, about how they're using IPFS as their primary file storing system. 

[Radicle & Monadic - Julian Arni](https://youtu.be/UCDHMKC8Seg) 
Julian tells us about his peer-to-peer collaboration platform Radicle and speaks about the benefits of decentralized systems. 

[World Brain - Oliver Sauter](https://youtu.be/e1YfSa2_Drc) 
We talk to Oliver about their decentralized search tool, Memex, and their peer-to-peer search engine for personal and collective knowledge, Worldbrain.io. 

[IPFS Force - Steven Li](https://youtu.be/KsMqesFi4Ac) 
And on top of all that, Steven joins us from Shanghai City to discuss his work on IPFS-Force.

Reply:
Zak - I'm moving these to their own post. They're so great and I feel like after all the awesomeness of the scifi fair they are getting a little lost down here...